### PR TITLE
Update README for AWS templates

### PR DESCRIPTION
**Pull request by mssachan**

### CODE REVIEW

File: config-templates/aws/aws-ebs-csi-driver/0.8.2/README.md [hunk 1]
```diff
@@ -1,131 +1,98 @@
...
+**Example `sat storage assignment create` command**
+
+```sh
+ibmcloud sat storage assignment create --name install-ebs --cluster-group <cluster-group> --configuration aws-ebs-conf
+```
+
+## Verifying your AWS EBS CSI Driver storage configuration is assigned to your clusters
+
+Provide steps to retrieve any driver pods, storage classes, or any other resources that are deployed with your storage configuration.
+
```
Comment (derekpoindexter) [inline config-templates/aws/aws-ebs-csi-driver/0.8.2/README.md:66]:
```suggestion

List the EBS driver pods in the `kube-system` namespace and verify that the status is `Running`.

```

File: config-templates/aws/aws-ebs-csi-driver/0.8.2/README.md [hunk 2]
```diff
@@ -1,131 +1,98 @@
...
+```
+$ oc get pods -n kube-system | grep ebs    
+ebs-csi-controller-5dc5468b9-d8c5c      6/6     Running   0          2d8h
+ebs-csi-controller-5dc5468b9-p2tw4      6/6     Running   0          2d8h
+ebs-csi-node-2vt9v                      3/3     Running   0          2d8h
+ebs-csi-node-7wccn                      3/3     Running   0          2d8h
+ebs-csi-node-rllvw                      3/3     Running   0          2d8h
+ebs-snapshot-controller-0               1/1     Running   0          2d8h
+```
+
```
Comment (derekpoindexter) [inline config-templates/aws/aws-ebs-csi-driver/0.8.2/README.md:76]:
```suggestion

List the EBS storage classes.

```

File: config-templates/aws/aws-efs-csi-driver/1.0.0/README.md [hunk 3]
```diff
@@ -1,112 +1,65 @@
...
+# AWS EFS CSI Driver
+
+AWS EFS CSI driver implements the CSI specification for container orchestrators to manage the lifecycle of Amazon EFS file systems.
+
+## Prerequisites
 
-### Pre-reqs 
 - Currently only *static provisioning* is supported. This means an [AWS EFS file system](https://docs.aws.amazon.com/efs/latest/ug/gs-step-two-create-efs-resources.html) needs to be created manually on AWS first. After that it can be mounted inside a container as a volume using the driver.
 
+## AWS EBS CSI Driver parameters & how to retrieve them
```
Comment (derekpoindexter) [inline config-templates/aws/aws-efs-csi-driver/1.0.0/README.md:9]:
```suggestion
## AWS EFS CSI Driver parameters & how to retrieve them
```

File: config-templates/aws/aws-efs-csi-driver/1.0.0/README.md [hunk 4]
```diff
@@ -1,112 +1,65 @@
...
+
+No parameters required for this template.
```
Comment (derekpoindexter) [inline config-templates/aws/aws-efs-csi-driver/1.0.0/README.md:11]: Are there any optional parameters that the user can specify? If so, we should list those here.
```suggestion
No parameters required for this template. 
```

File: config-templates/aws/aws-efs-csi-driver/1.0.0/README.md [hunk 5]
```diff
@@ -1,112 +1,65 @@
...
+## Creating the storage assignment
+
+**Example `sat storage assignment create` command**
+
+```
+ibmcloud sat storage assignment create --name install-efs --cluster-group <cluster-group> --configuration aws-efs-conf
+```
+
+## Verifying your AWS EFS CSI Driver storage configuration is assigned to your clusters
+
```
Comment (derekpoindexter) [inline config-templates/aws/aws-efs-csi-driver/1.0.0/README.md:38]:
```suggestion
List the EFS driver pods in the `kube-system` namespace and verify that the status is `Running`.
```

File: config-templates/aws/aws-efs-csi-driver/1.0.0/README.md [hunk 6]
```diff
@@ -1,112 +1,65 @@
...
+```
+$ oc get pods -n kube-system | grep efs    
+efs-csi-node-4gkzx                      3/3     Running   0          2d22h
+efs-csi-node-r8g5d                      3/3     Running   0          2d22h
+efs-csi-node-td4wc                      3/3     Running   0          2d22h
+```
+
```
Comment (derekpoindexter) [inline config-templates/aws/aws-efs-csi-driver/1.0.0/README.md:45]:
```suggestion

List the EFS storage classes.

```

File: config-templates/aws/aws-ebs-csi-driver/0.8.2/README.md [hunk 7]
```diff
@@ -1,131 +1,98 @@
...
 - Dyanamic Provisioning
 - Volume Resizing
 - Volume Snapshot 
 
-### Pre-reqs
-#### Planning consideration for Infra Admin
--  For better IOPS and throughput, use instance built on *Nirto System*. Refer to 
+## Prerequisites
+**Planning consideration for Infra Admin**
+-  For better IOPS and throughput, use instance built on *Nitro System*. Refer to
```
Comment (derekpoindexter) [inline config-templates/aws/aws-ebs-csi-driver/0.8.2/README.md:12]:
```suggestion
-  For better IOPS and throughput, create an instance that uses *Nitro System*. Refer to 
```

**Review by derekpoindexter — APPROVED** [review]

File: config-templates/aws/aws-ebs-csi-driver/0.8.2/README.md [hunk 8]
```diff
@@ -1,131 +1,98 @@
...
+**Planning consideration for Infra Admin**
+-  For better IOPS and throughput, use instance built on *Nitro System*. Refer to 
 https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-volume-types.html
 
-#### Planning consideration for Location Admin
+**Planning consideration for Location Admin**
 - The driver requires IAM permission to talk to Amazon EBS to manage the volume on user's behalf. Create an IAM user with proper permission and get *AWS Access Keys*. Refer to https://docs.aws.amazon.com/powershell/latest/userguide/pstools-appendix-sign-up.html
 
 - Volume type *io2* requires iopsPerGB, default is set to *10 IOPS/GB*.
   To set different value for iopsPerGB, refer to https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-volume-types.html
```
Comment (derekpoindexter) [inline config-templates/aws/aws-ebs-csi-driver/0.8.2/README.md:19]: Let's put this sentence in the parameter table with the `iops` parameter - and include a statement that says they should pass the IOPs value that is required for the volume that type that they want to use.

**Comment by derekpoindexter** [comment]
@mssachan - I've approved this pr, but I left some commit suggestions and requested additional information in some areas - let me know if you have any questions.

File: config-templates/aws/aws-efs-csi-driver/1.0.0/README.md (hunk 4 above)
Comment (mssachan) [reply config-templates/aws/aws-efs-csi-driver/1.0.0/README.md:11]: There is no optional parameters required for AWS EFS.

File: config-templates/aws/aws-ebs-csi-driver/0.8.2/README.md (hunk 8 above)
Comment (mssachan) [reply config-templates/aws/aws-ebs-csi-driver/0.8.2/README.md:19]: Moved to desciption column. Also `iops-per-gb` is required only by `sat-aws-block-gold` storageclass (volume type: io2). So I added note for same.
`sat-aws-block-gold storageclass (Volume type io2) requires iopsPerGB, default is set to 10 IOPS/GB. To set different value for iopsPerGB, refer to https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-volume-types.html
`

File: config-templates/aws/aws-ebs-csi-driver/0.8.2/README.md [hunk 9]
```diff
@@ -1,131 +1,98 @@
...
+ibmcloud sat storage template get --name aws-ebs-csi-driver --version 0.8.2
+```
+
+**AWS EBS CSI Driver parameters**
+
+| Parameter | Required? | Description | Default value if not provided |
+| --- | --- | --- | --- |
+| `aws-access-key` | Required | This is a required parameter. | N/A |
+| `aws-secret-access-key` | Required | This is a required parameter. | N/A | |
+| `iops-per-gb` | Optional | This is an optional parameter.  | 10 |
```
[the remark this answers is not in the document]
Comment (mssachan) [reply config-templates/aws/aws-ebs-csi-driver/0.8.2/README.md]:
```
Provisioned IOPS SSD (io1 and io2) volumes are designed to meet the needs of I/O-intensive workloads, particularly database workloads, that are sensitive to storage performance and consistency. Provisioned IOPS SSD volumes use a consistent IOPS rate, which you specify when you create the volume, and Amazon EBS delivers the provisioned performance 99.9 percent of the time
```
https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-volume-types.html#EBSVolumeTypes_piops

File: config-templates/aws/aws-efs-csi-driver/1.0.0/README.md [hunk 10]
```diff
@@ -1,112 +1,67 @@
-## How to use the template to deploy aws-efs-csi-driver on a Satellite Cluster?
+# AWS EFS CSI Driver
+
+AWS EFS CSI driver implements the CSI specification for container orchestrators to manage the lifecycle of Amazon EFS file systems.
+
+## Prerequisites
 
-### Pre-reqs 
 - Currently only *static provisioning* is supported. This means an [AWS EFS file system](https://docs.aws.amazon.com/efs/latest/ug/gs-step-two-create-efs-resources.html) needs to be created manually on AWS first. After that it can be mounted inside a container as a volume using the driver.
```
Comment (derekpoindexter) [inline config-templates/aws/aws-efs-csi-driver/1.0.0/README.md:7]:
```suggestion
- Currently only static provisioning is supported. Before you can create a Satellite storage configuration, you must manually provision an [AWS EFS file system](https://docs.aws.amazon.com/efs/latest/ug/gs-step-two-create-efs-resources.html). After you create your AWS EFS instance, you can create a Satellite storage configuration which references your EFS and can be mount your EFS inside a container.
```

**Review by nkkashyap — APPROVED** [review]
Looks good

**Pull request merged**